landingPage/landingPage.css

```diff
@@ -0,0 +1,217 @@
+
+*
+{
+    font-family: 'Lato', sans-serif;
+}
+
+
+body
+{
+margin: 0;
+padding: 0;
+    
+
+}
+
+:root
+{
+    --primary-color: #63DCCD;
+    --secondary-color: #266350;
+}
+
+.logo-component
+{
+
+    height: 50px;
+    width: 60px;
+}
+
+.navigation-menu
+{
+    padding: 1.5rem 1rem;
+    /* width: 100%; */
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    align-items: center;
+    /* border-bottom: 2px solid var(--primary-color); */
+}
+
+.left-side-nav
+{
+    display: flex;
+
+    justify-content: space-between;
+    align-items: center;
+}
+
+.right-side-nav
+{
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
```

Comment:
in the nav container use have used space-evenly,also in the child elements also justify-content has been set to space between.so you can just give margin right auto to left part of nav and margin left auto to right part of nav and keep the parent nav container as flex without giving it any justify content.Or I guess even if you have given space between in parent container or any other similar kind of space property ,you don't need to mention in child again.